drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt

```diff
@@ -112,23 +116,34 @@ class DropInViewModel(
         amount = dropInConfiguration.amount
     }
 
-    val showPreselectedStored = paymentMethodsApiResponse.storedPaymentMethods?.any { it.isEcommerce } == true &&
-        dropInConfiguration.showPreselectedStoredPaymentMethod
+    fun getPaymentMethods(): List<PaymentMethod> {
+        return paymentMethodsApiResponse?.paymentMethods.orEmpty()
+    }
+
+    fun getStoredPaymentMethods(): List<StoredPaymentMethod> {
+        return paymentMethodsApiResponse?.storedPaymentMethods.orEmpty()
+    }
 
-    val preselectedStoredPayment
-        get() = paymentMethodsApiResponse.storedPaymentMethods?.firstOrNull {
+    fun showPreselectedStored(): Boolean {
```

Comment:
This name implies that something will be shown, so I would call it `shouldShowPreselectedStored`